src/DependencyInjection/DI/src/ServiceLookup/ILEmit/ILEmitResolverBuilder.cs

```diff
@@ -276,42 +270,7 @@ private void AddCacheKey(ILEmitResolverBuilderContext argument, ServiceCacheKey
             argument.Generator.Emit(OpCodes.Newobj, CacheKeyCtor);
         }
 
-
-        private Func<ServiceProviderEngineScope, object> BuildType(ServiceCallSite callSite)
-        {
-            // We need to skip visibility checks because services/constructors might be private
-            var dynamicMethod = new DynamicMethod("ResolveService",
-                attributes: MethodAttributes.Public | MethodAttributes.Static,
-                callingConvention: CallingConventions.Standard,
-                returnType: typeof(object),
-                parameterTypes: new [] {typeof(ILEmitResolverBuilderRuntimeContext), typeof(ServiceProviderEngineScope) },
-                owner: GetType(),
-                skipVisibility: true);
-
-            var info = ILEmitCallSiteAnalyzer.Instance.CollectGenerationInfo(callSite);
-            var runtimeContext = GenerateMethodBody(callSite, dynamicMethod.GetILGenerator(info.Size), info);
-
-#if SAVE_ASSEMBLY
```

Comment:
Do we have a test for this? Apparently this should have been SAVE_ASSEMBLIES even before this change.

Reply:
This is a feature I only use when debugging code IL code generation it's always excluded from shipping builds.